app/decorators/user_decorator.rb

```diff
@@ -20,7 +20,8 @@ def age_info
   def interests_info
     if points_of_interests.any?
       points_of_interests.each do |poi|
-        poi.points
+       return "#{poi.name} type-of
```

Comment:
I think the return in needed because, without it, method return us all information about object, not same asked fields